app/src/main/java/com/sunlightlabs/android/congress/fragments/AlertFragment.java

```diff
@@ -64,24 +64,24 @@ public Dialog about(LayoutInflater inflater) {
 		View aboutView = inflater.inflate(R.layout.about, null);
 
 		Spanned about1 = Html.fromHtml(
```

Comment:
I updated the About fragment text to use current links, to cite Pro Publica, to remove the Sunlight logo, and to cite me (while retaining a grateful cite to Sunlight for their years of support).